docs/consensus/migration-to-pos.md

```diff
@@ -0,0 +1,20 @@
+---
+id: migration-to-pos
+title: Migration from PoA to PoS
+---
+
+## Overview
+
+This section guides you through migration from PoA to PoS for the running cluster without reset of blockchain.
```

Comment:
I would just rephrase the last section of this sentence:

`This section guides you through the migration from PoA to PoS IBFT modes, and vice versa, for a running cluster - without the need to reset the blockchain.`

Reply:
Fixed at c54b7deb2c83464f83a348a110ea278cef3f0798